apps/docs/src/.vitepress/components/OnyxColorPalettes.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts" setup>
+import { useData } from "vitepress";
+import ColorPalette from "./ColorPalette.vue";
+
+const { isDark } = useData();
+</script>
+
+<template>
+  <div class="palettes">
+    <div class="themes">
+      <Badge
```

Comment:
Two comments about the Theme Toggle:
1. From a a11y and semantic html perspective we should only use the respective elements for user interactions (buttons, inputs, anchors etc.) and not add click and keyup handlers to arbitrary elements.
2. IMHO we shouldn't toggle the theme of the complete page to view the respective color palettes. My eyes hurt everytime I toggle between the light and dark theme 😄  Maybe we can limit the toggle to only the palette content?

Reply:
The points don't need to be adressed in this PR :)

Reply:
1. fixed
2. The color palettes are using the CSS variables which only exist if the application is in light/dark theme. I don't know if there is a technical way to show the colors otherwise because the CSS variables do not exists then. Imho #179 fixes this

Reply:
Okay, thanks for updating the ticket! I extended the description to also include it for `sit-onyx`